src/build.py

```diff
@@ -68,7 +68,11 @@ def render(template, dest, **kwargs):
     template = environment.get_template(template)
     text = template.render(**kwargs)
 
-    f = file(dest, 'wb')
+    if sys.version_info[0] < 3:
+        f = file(dest, "wb")
+    else:
+        f = open(dest, 'wb')
+
```

Comment:
I don't really know about 2, but I think you are 99% right. My way was "not to brake", so I just turned off my mind in version 2 branch of this if statement :innocent: